platform/macos/common.cmake

```diff
@@ -29,6 +30,57 @@ install(
     WORLD_READ             WORLD_EXECUTE 
 )
 
+install(
+  FILES
+    "${OSQUERY_DATA_PATH}/control/Info.plist"
+    "${OSQUERY_DATA_PATH}/control/PkgInfo"
+    "${OSQUERY_DATA_PATH}/control/embedded.provisionprofile"
+
+  DESTINATION
+    "/opt/osquery.app/Contents"
+
+  COMPONENT
+    osquery
+)
+
+install(
+  FILES
+    "${OSQUERY_DATA_PATH}/opt/bin/osqueryctl"
+
+  DESTINATION
+    "/opt/osquery.app/Contents/Resources"
+
+  COMPONENT
+    osquery
+
+  PERMISSIONS
+    OWNER_READ OWNER_WRITE OWNER_EXECUTE
+    GROUP_READ             GROUP_EXECUTE
+    WORLD_READ             WORLD_EXECUTE 
+)
+
+execute_process(
+  COMMAND "${CMAKE_COMMAND}" -E create_symlink "/opt/osquery.app/Contents/MacOS/osqueryd" osqueryi
+  WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}"
+)
+
+execute_process(
+  COMMAND "${CMAKE_COMMAND}" -E create_symlink "/opt/osquery.app/Contents/Resources/osqueryctl" osqueryctl
+  WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}"
+)
+
+install(
+  FILES
+    "${CMAKE_CURRENT_BINARY_DIR}/osqueryi"
+    "${CMAKE_CURRENT_BINARY_DIR}/osqueryctl"
+  
+  DESTINATION
+    "/usr/local/bin/"
```

Comment:
Help me understand this section. Are we still installing things into usr/local?

Reply:
I think we should not.

Reply:
These are the symlinks installed in `/usr/local/bin`:

```
usr
  └── local
      └── bin
          ├── osqueryctl -> /opt/osquery.app/Contents/Resources/osqueryctl
          └── osqueryi -> /opt/osquery.app/Contents/MacOS/osqueryd
```

Reply:
Let's keep it for this PR and follow up with a second if we do want to remove these (I think we do). The approach I recommend on macOS is to start installing a `/etc/paths.d/osquery` file with our `/opt/osquery.app/...` path.